documentation/partner_backend.md

```diff
@@ -104,23 +104,30 @@ To obtain a token from the Latch Auth0 App, the following steps need to be execu
 	* `email_verified`: Unused field.
 	* `_id`: Unused field.
 
-	In case of an error, the API will return an HTTP 400 and a status in the "error_description" field:
+	In case of an error, the API will return the following HTTP error codes and detailed information in the `error_description` field:
 	
-	* `BAD_REQUEST`: Missing "email" parameter.
+	* HTTP 400
+		* `error="bad.email"`: Missing "email" parameter.
 	
-		⇒ Check the API request to make sure all fields have been populated.
+			⇒ Check the API request to make sure all fields have been populated.
 		
-	* `UNAUTHORIZED`: Email account is not authorized or doesn't exist.
+		* `error="extensibility_error" error_description="UNAUTHORIZED"`: Email account is not authorized or doesn't exist.
 
-		⇒ Check if the email has been configured in Manager Web.
+			⇒ Check if the email has been configured in Manager Web.
```

Comment:
Change `Manager Web` to `Mission Control`

Reply:
Also this check should only help if the user was invited in Mission Control. If invited via the UserKit APIs, this does not apply. I think it makes sense to add some clarification here.

Reply:
Updated all references to Manager Web with Mission Control, and added an explanation that it could have been configured via Mission Control or programmatically via UserKit.